src/server.ts

```diff
@@ -227,6 +227,8 @@ const get_metadata = (ecosystem, name, version) => {
                 }
                 options['headers'] = {
                     'Authorization' : 'Bearer ' + config.api_token,
+                    'security-vendor' : custom_header.security_vendor,

```

Comment:
You shouldn't explicitly retrieve the custom headers fields, it defeats the purpose of passing it as JSON.You _should_ concat those.

`options['headers'].concat(customHeader);`